src/components/NonProfitApply/NonProfitApply.js

```diff
@@ -17,15 +17,18 @@ import useNonprofit from "../../hooks/use-nonprofit";
 import { useAuth0 } from "@auth0/auth0-react";
 import Stack from "@mui/material/Stack";
 import Alert from "@mui/material/Alert";
-import AlertTitle from "@mui/material/AlertTitle";
-import Typography from "@mui/material/Typography";
+
+
 import Moment from 'moment';
 import Image from 'next/image';
 import PlaceIcon from '@mui/icons-material/Place';
 import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
 import WarningIcon from '@mui/icons-material/Warning';
 import Autocomplete from "@mui/material/Autocomplete";
 
+import ReactRecaptcha3 from 'react-google-recaptcha3';
```

Comment:
This was the easiest v3 RECAPTCHA solution, doesn't require the "pick the boats" things for people, and uses algos to determine if bots are submitting stuff behind the scenes.